packages/ui-components/src/components/relative-time-picker/relative-time-picker.helper.ts

```diff
@@ -62,6 +62,11 @@ export const buildRelativeAmountOfUnitsConfigRange = (option: IRelativeTimePicke
 	const nowDateTime = newTimezoneDate(timeZone);
 	const { amount, unit } = option.startDate;
 	const calculatedDate = calculateDate(nowDateTime, amount, unit);
+
+	if (amount === 0) {
```

Comment:
🔨

Reply:
This was a really specific case that only happens when we are comparing a relative date to now and the amount of units we are removing is 0. For example, considering "Today" example, we want the time from the start of the day until the current timestamp. Considering the config file: 

	`label:` `'Today',`
	`value:` `'today',`
	`comparisonConfig:` `ERelativeTimeComparisonConfig.RelativeAmountOfUnits,`
	`startDate:` {
		`amount:` `0,`
		`unit:` `'days'`
	`}`

We are using the `calculateDate` to add amounts of units to the current date. Therefore, when the amount is negative, we take x units from the current timestamp. But here in this specific case, we are removing 0 days from the current timestamp. That being said, we are actually asking for the current timestamp which will be a bad result because the interval will be from now until now. 

My concern here with changing the calculate date function is that we will be inserting unexpected behaviour on this util. We can see it both ways and maybe the suggestion that Diogo gave the other day would be a better alternative for this case that would consist in adding in the config file maybe an enum to determine if we want the start of or the end of unit like the example bellow:

	`startDate:` {
		`amount:` `0,`
		`unit:` `'days'`
                `limit`: `EUnitRange.StartOf` || `EUnitRange.EndOf`
	`}`

But this change to the config file would only make sense to cover this edge case since this does change does not apply to the other `comparisonConfig` options.